### ISSUE TITLE
add_or_update_assignments action is missing along with others

### ISSUE THREAD

**Issue by wstewartii** [letsbondiway/fastlane-plugin-airwatch_workspaceone#1]
The documentations says that the following actions should be available.

```
fastlane action delete_previous_versions
fastlane action retire_specific_version
fastlane action delete_specific_version
fastlane action add_or_update_assignments
fastlane action unretire_all_versions
```

but I only have these options

```
[09:21:19]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
+---------------------------------------+---------+---------------------------------------+
|                                      Used plugins                                       |
+---------------------------------------+---------+---------------------------------------+
| Plugin                                | Version | Action                                |
+---------------------------------------+---------+---------------------------------------+
| fastlane-plugin-airwatch_workspaceone | 1.0.2   | retire_previous_versions deploy_build |
+---------------------------------------+---------+---------------------------------------+
```

The `add_or_update_assignments` action is the one that is of the most interest to me.

**Comment by wstewartii** [comment]
This was resolved by updating to version 2.0.0

**Comment by letsbondiway** [comment]
@wstewartii Glad to know that. Just in case if you are running this on a CI system, please ensure to run [sudo] bundle install as the first step in CI after checking your code out.
